content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst

```diff
@@ -10,98 +10,94 @@ In Odoo, this process can be conducted manually— by warehouse employees physic
 products— or automatically through the database.
 
 .. seealso::
-   :ref:`Inventory valuation configuration <inventory/inventory_valuation_config>`
+   :ref:`Inventory valuation methods <inventory/management/reporting/inventory_valuation_config>`
 
-Understand the basics of inventory valuation
-============================================
+Inventory valuation basics
+==========================
 
 In order to understand how moving products in and out of stock affects the company's overall value,
 consider the following product and stock moves scenario below.
 
 Receive a product
 -----------------
 
-For example, consider an physical product, a simple *table*, which is categorized as `Office
+For example, consider a physical product, a simple *table*, which is categorized as `Office
 Furniture` in the product form's :guilabel:`Product Category` field.
```

Comment:
Also with this text - if this is a selected value from a dropdown it should be guilabel, otherwise user input as literal
```suggestion
For example, consider a physical product, a simple *table*, which is categorized as :guilabel:`Office
Furniture` in the product form's :guilabel:`Product Category` field.
```